src/components/HiddenText/HiddenText.scss

```diff
@@ -0,0 +1,6 @@
+@import "../../styles/themes.scss";
+@import "../../styles/typography.scss";
+
+.hidden-text--wrapper {
+  display: none;
```

Comment:
this could be problematic for screen readers - 
https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/